cabal-ghc901.project

```diff
@@ -47,6 +47,7 @@ constraints:
 -- although we are not building all plugins cabal solver phase is run for all packages
 -- this way we track explicitly all transitive dependencies which need support for ghc-9
 allow-newer:
+    *:ghc-bignum,
```

Comment:
I'd have expected this to be in `cabal-ghc921.project`, what's the issue? Worth a comment.

Reply:
I probably just met alike situation in `hnix`

It may be because `ghc-bignum` [`1.1` is not present](https://hackage.haskell.org/package/ghc-bignum) on Hackage.
The upstream tracking link currently is: https://gitlab.haskell.org/ghc/ghc/-/issues/19632#note_400751

Versions of `ghc-bignum` used as a [boot](https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history) library:
`ghc 8.x` uses `1.0` 
`ghc 9.0` uses `1.1` 
`ghc 9.2` uses `1.2`

So use of `9.0`, with some other bounds - should solve into `1.1`, but that version is not on Hackage.

Reply:
It could be an accidental addition. Rather than guessing, let's try removing it.

Reply:
> `ghc 9.0` uses `1.1`

It's worse than that: 9.0.1 uses 1.0, but 9.0.2 uses 1.1. Anyway, `hashable` has been recently revised to allow `ghc-bignum-1.1`, so I think this `allow-newer` can be safely removed.

Reply:
At least a few package-scoped `X:ghc-bignum` allow-newers would be nicer than a blanket one.

Reply:
ghc-bignum is a boot library, so the solver has no choice in which version it picks.

Reply:
`ghc-bignum` is no longer in this file.